README.md

```diff
@@ -2,22 +2,19 @@
 [![Repository](https://img.shields.io/badge/github-dusk--bls12_381-blueviolet?logo=github)](https://github.com/dusk-network/bls12_381)
 [![Documentation](https://img.shields.io/badge/docs-dusk--bls12_381-blue?logo=rust)](https://docs.rs/dusk-bls12_381/)
 
-**THIS CRATE IS A FORK OF [https://github.com/zkcrypto/bls12_381](https://github.com/zkcrypto/bls12_381/) where the Dusk-Network team has added a variety of tools required by other libraries built on the top of this one.
-The 99% of the library stills being done by @ebfull and you SHOULD NOT use this one unless you need a specific tool that we've 
-implemented and it's not on the original library.**
+> :warning: THIS CRATE IS A FORK OF [https://github.com/zkcrypto/bls12_381](https://github.com/zkcrypto/bls12_381/): The Dusk team has added a variety of tools required for its own uses on the top of the original library. You SHOULD NOT use this library unless you need a specific tool that we've implemented and is not available in the original.
 
-## Extra tools added to the original bls12_381 lib:
+## Extra tools added to [bls12_381](https://github.com/zkcrypto/bls12_381/) lib:
 - Add serde support for every single data structure in the crate that is exported.
 - Add various multiscalar_mul algorithms.
-- Use `std & endo` as a default feature.
 - Impl Iter Sum & Product for Scalar.
-- Implement random for Scalar.
+- Implement hash for Scalar.
 - Implement XOR & AND for Scalar.
 - Add base_4 conversion fn (no longer required).
 - Impl Ord & PartialOrd for Scalar.
 - Implement w_naf_scalar_mul (71% faster than the original double-and-add impl).
 - Implement a reduce function wrapper for Scalar.
-- Expose some Scarlar-related Constants as public.
+- Expose some Scalar-related constants as public.
 
```

Comment:
Also:
- Add our own implementation of hashing to a `Scalar`

Reply:
@moCello Added!